rest/data/interface.go

```diff
@@ -139,7 +139,7 @@ type Connector interface {
 	SetBannerTheme(string, *user.DBUser) error
 	// SetAdminBanner sets set the service flags in the system-wide settings document
 	SetServiceFlags(admin.ServiceFlags, *user.DBUser) error
-	RestartFailedTasks(time.Time, time.Time, string, bool) (*restModel.RestartTasksResponse, error)
+	RestartFailedTasks(time.Time, time.Time, string, bool, bool, bool) (*restModel.RestartTasksResponse, error)
```

Comment:
should we also use the new type in this API?

Reply:
done